qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/lifecycle/viewmodel/DropInLocationViewModel.kt

```diff
@@ -43,8 +43,8 @@ class DropInLocationViewModel : ViewModel() {
         override fun onAttached(mapboxNavigation: MapboxNavigation) {
             val locationEngine = mapboxNavigation.navigationOptions.locationEngine
             locationEngine.getLastLocation(object : LocationEngineCallback<LocationEngineResult> {
-                override fun onSuccess(result: LocationEngineResult) {
-                    result.lastLocation?.let {
+                override fun onSuccess(result: LocationEngineResult?) {
+                    result?.lastLocation?.let {
```

Comment:
it seems like a breaking API change, doesn't it?

Reply:
Yes, it is. As mentioned above and in the changelog, that will be fixed with the next update of Common SDK (before stable).